src/app/metadata-schema-form/metadata-form/metadata-form.component.ts

```diff
@@ -80,7 +80,7 @@ export class MetadataFormComponent implements OnInit {
   }
 
   tabIsVisible(tab: MetadataFormTab): boolean {
-    if (this.config.viewMode && this.config.removeEmptyFields && tab.items.length === 1) {
+    if (this.config.viewMode && this.config.hideEmptyFields && tab.items.length === 1) {
```

Comment:
What does this predicate check?
Have you considered extracting a function here?

Reply:
My only change here was refactoring the name.
The tabIsVisible method does look it is in need of some simplification but I think it's out of scope for the changes I was making.

Reply:
That is fair.
Thanks.